README.md

```diff
@@ -12,7 +12,7 @@ This library provides efficient and flexible implementations of various halo2-fr
 
 The implementations were originally ported from [matterlabs/pairing](https://github.com/matter-labs/pairing/tree/master/src/bn256) and [zkcrypto/bls12-381](https://github.com/zkcrypto/bls12_381), but have been extended and optimized to cover a broader set of curves and use cases. Since its initial release, the library has expanded to include additional curves, along with the following features:
 
-* `secp256k1`, `secp256r1`, and `grumpkin` curves, enhancing its usability across a range of cryptographic protocols.
+* `secp256k1`, `secp256r1`, `pluto`, `eris` and `grumpkin` curves, enhancing its usability across a range of cryptographic protocols.
```

Comment:
I think it is better to use the `pluto-eris` or `pluto/eris` for the names of new curves.
The reason is it delivers more clear meaning. (like [here](https://github.com/daira/pluto-eris?tab=readme-ov-file#plutoeris-supporting-evidence)
What do you think? @CPerezz

Reply:
> I think it is better to use the `pluto-eris` or `pluto/eris` for the names of new curves. The reason is it delivers more clear meaning. (like [here](https://github.com/daira/pluto-eris?tab=readme-ov-file#plutoeris-supporting-evidence))
 What do you think? @CPerezz

Reply:
I'd rather keep it as is. Since they're 2 different curves. We follow the same rationale for the secp cycle indeed.